src/fsharp/utils/FileSystem.fsi

```diff
@@ -9,18 +9,22 @@ open System.Reflection
 open System.Text
 open System.Runtime.CompilerServices
 
-exception IllegalFileNameChar of string * char
+exception internal IllegalFileNameChar of string * char
```

Comment:
Same comment as the last pr it was in, is it possible that a compiler service developer does or could reasonably depend on this exception?